tests/test_iotedgemodule.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import os
+
+root_dir = os.getcwd()
+module_name = "filterModule"
+
+@pytest.fixture(scope="module",params=["requirements.txt","module.json","main.py","Dockerfile.arm32v7","Dockerfile.amd64.debug","Dockerfile.amd64",".gitignore"])
+def test_CheckFileExists(request):
```

Comment:
Minor: Python uses the `test_check_file_exist` style

Reply:
thanks